src/py42/sdk/queries/query_filter.py

```diff
@@ -50,10 +50,23 @@ def create_in_range_filter_group(term, start_value, end_value):
     return create_filter_group(filter_list, u"AND")
 
 
+def create_within_the_last_filter_group(term, value):
+    filter_list = [create_query_filter(term, "WITHIN_THE_LAST", value)]
+    return create_filter_group(filter_list, "AND")
+
+
 def create_filter_group(query_filter_list, filter_clause):
     return FilterGroup(query_filter_list, filter_clause)
 
 
+def filter_attributes(cls):
```

Comment:
this is returning the _names_ of the attributes, we would want to return the _values_ of the attributes. The idea here is to be able to do something like loop through every file category to build a list of valid file categories.

Reply:
Changed.